firmware/lucidgloves-firmware/PCA9685Servo.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <Adafruit_PWMServoDriver.h>
+
+Adafruit_PWMServoDriver pwm_board_0 = Adafruit_PWMServoDriver(PWM_Board_0_I2C_ADDRESS, Wire);
+
+
+int Initialize_PCA9685_Board()
+{
+  pwm_board_0.begin();
+  pwm_board_0.setOscillatorFrequency(25000000);
+  pwm_board_0.setPWMFreq(PWM_Board_0_PWM_FREQUENCY);  // Analog servos usually run at ~50 Hz updates
+  Serial.println("PCA9685 Board Initialized ");
+}
+
+
+#define MIN_PULSE_WIDTH ServoMin_uS   //Sets the MIN_PULSE_WIDTH setting used by smooth stepping to your configured ServoMin (Default in esp32servo is 500)
+#define MAX_PULSE_WIDTH ServoMax_uS   //Sets the Max_PULSE_WIDTH setting used by smooth stepping to your configured ServoMax (Default in esp32servo is 2500)
```

Comment:
Can all the function and method definitions be in a .cpp file insted of a header?